doc/intro.md

```diff
@@ -30,8 +30,8 @@ example, given an instance of the `Account` type assigned to the variable
 
 ```proto
 has(account.user_id) || has(account.gaia_id)    // true if either one is set
-size(account.emails) > 0                        // true if emails is non-empty
-matches(account.phone_number, "[0-9-]+")        // true if number matches regexp
+account.emails.size() > 0                       // true if emails is non-empty
```

Comment:
Please add the 4 receiver-style size() declarations to the list of standard definitions.

Reply:
I've basically replaced the global existing definitions with the receiver-style ones. That's been the preference internally for quite some time, but I didn't realize this wasn't reflected publicly.